README.md

```diff
@@ -137,8 +137,10 @@ components in the Pulsar ecosystem, including connectors, adapters, and other la
 - Maven 3.6.1+
 - zip
 
-> Note: this project includes a [Maven Wrapper](https://maven.apache.org/wrapper/) that can be used instead of a system installed Maven.
-> Use it by replacing `mvn` by `./mvnw` on Linux and `mvnw.cmd` on Windows in the commands below.
+> Notes:
```

Comment:
```suggestion
> **Note**:
```

Trying to make use of GitHub rendering like:

<img width="581" alt="image" src="https://user-images.githubusercontent.com/18818196/200887060-c93197b8-38be-4a74-ba0b-4ff355c78f28.png">